Jenkinsfile

```diff
@@ -64,7 +64,9 @@ node ('master') {
 
             // Run the tests
             stage("Run Tests") {
-                sh './bin/run_docker_test tests/unit-poet2.yaml'
+                sh './bin/run_docker_test tests/unit-poet.yaml'
+                sh './bin/run_docker_test tests/unit-ias-client.yaml'
```

Comment:
No hard rules, we could maintain single file with multiple services vs multiple docker-compose files.
I'm ok with either approach.

Thanks for this change 👍